### PR TITLE
Fix empty glob in boost.thread

### PR DESCRIPTION
**Pull request by hofbi**
Needs skip-url-check

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @wep21, @Vertexwahn, modules you maintain (boost.thread) have been updated in this PR. Please review the changes.

**Comment by Vertexwahn** [comment]
@bazel-io skip_check unstable_url

**Comment by Vertexwahn** [comment]
@hofbi See my last comment - you can get the skip-url-stability-check label by posting this comment

**Review by Vertexwahn — DISMISSED** [review]

**Review by bazel-io — DISMISSED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

File: modules/boost.thread/1.83.0.bcr.1/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+bazel_dep(name = "boost.iterator", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.lexical_cast", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.move", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.optional", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.predef", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.preprocessor", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.smart_ptr", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.static_assert", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.system", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.throw_exception", version = "1.83.0.bcr.1")
```
Comment (wep21) [inline modules/boost.thread/1.83.0.bcr.1/MODULE.bazel:32]:
```suggestion
bazel_dep(name = "boost.throw_exception", version = "1.83.0.bcr.1")
bazel_dep(name = "boost.tuple", version = "1.83.0.bcr.1")
```

**Comment by wep21** [comment]
@bazel-io skip_check unstable_url

File: modules/boost.thread/1.83.0.bcr.1/source.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "integrity": "sha256-zIFNyvMDzgajwBoU+gvJoAGXbEPKHOoeATxxNeoxB5s=",
+    "strip_prefix": "thread-boost-1.83.0",
+    "url": "https://github.com/boostorg/thread/archive/refs/tags/boost-1.83.0.tar.gz",
+    "patch_strip": 0,
+    "overlay": {
+        "BUILD.bazel": "sha256-ec85Zp/2Y4kqR7RN2bp0qA1UQmmAnnUZKM718nKtltg=",
+        "MODULE.bazel": "sha256-hORLxlKDt9y/13S5+laGou1oOQnNTeMnUzIWdl1wNQc=",
```
Comment (wep21) [inline modules/boost.thread/1.83.0.bcr.1/source.json:8]:
```suggestion
        "MODULE.bazel": "sha256-LF2hR/JRsMevLwhhR3Jxh47TZCSO3YXG6TF1z25fmM4=",
```

**Comment by Vertexwahn** [comment]
Seems that Bazel 8 requires rules_cc...

File: modules/boost.thread/1.83.0.bcr.1/overlay/test/MODULE.bazel [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+bazel_dep(name = "boost.interprocess", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.test", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.thread")
+
+local_path_override(
+    module_name = "boost.thread",
+    path = "..",
+)
+
+bazel_dep(name = "boost.variant", version = "1.83.0.bcr.1")
```
Comment (wep21) [inline modules/boost.thread/1.83.0.bcr.1/overlay/test/MODULE.bazel:11]:
```suggestion
bazel_dep(name = "boost.variant", version = "1.83.0.bcr.1")
bazel_dep(name = "rules_cc", version = "0.0.14")
```

Comment (wep21) [reply modules/boost.thread/1.83.0.bcr.1/overlay/test/MODULE.bazel:11]: also needs sha update

File: modules/boost.thread/1.83.0.bcr.1/overlay/test/MODULE.bazel [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+bazel_dep(name = "rules_cc", version = "0.0.14")
```
Comment (wep21) [inline modules/boost.thread/1.83.0.bcr.1/overlay/test/MODULE.bazel:12]:
```suggestion
bazel_dep(name = "rules_cc", version = "0.0.14")
bazel_dep(name = "platforms", version = "0.0.10")
```

**Review by wep21 — APPROVED** [review]

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**